api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -344,6 +344,9 @@ const (
 	// DisableIgnitionServerAnnotation controls skipping of the ignition server deployment.
 	DisableIgnitionServerAnnotation = "hypershift.openshift.io/disable-ignition-server"
 
+	// KubeAPIServerGoAwayChance allows the --goaway-chance parameter of the kube-apiserver to be overridden from its default of 0
+	KubeAPIServerGoAwayChance = "hypershift.openshift.io/goaway-chance"
```

Comment:
Same comment about the name.

Reply:
Updated to `hypershift.openshift.io/kube-apiserver-goaway-chance`